pkg/plugin/driver/job/job.go

```diff
@@ -186,15 +186,13 @@ func (p *Plugin) Monitor(kubeclient kubernetes.Interface, _ []v1.Node, resultsCh
 // Cleanup cleans up the k8s Job and ConfigMap created by this plugin instance
 func (p *Plugin) Cleanup(kubeclient kubernetes.Interface) {
 	p.cleanedUp = true
-	gracePeriod := int64(1)
```

Comment:
removed because I actually want the default grace period of 30 seconds to give our pods enough time to react. See: https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods for subtlety.